Lambdas/GHOAuth/OAuthLambda.swift

```diff
@@ -78,19 +85,13 @@ struct GHOAuthHandler: LambdaHandler {
             throw OAuthLambdaError.invalidPublicKey
         }
 
-        let privateKeyString = try await secretsRetriever.getSecret(arnEnvVarKey: "ACCOUNT_LINKING_OAUTH_FLOW_PUB_KEY")
-        guard let privateKey = try? ECDSAKey.private(pem: privateKeyString.value) else {
-            throw OAuthLambdaError.invalidPublicKey
-        }
+        let signer = JWTSigner.es256(key: publicKey)
 
-        let signers = JWTSigners()
-        signers.use(.es256(key: privateKey))
-        signers.use(.es256(key: publicKey))
-
-        guard (try? signers.verify(state, as: GHOAuthPayload.self)) != nil else {
-            logger.trace("State parameter verification failed")
+        guard let payload = try? signer.verify(state, as: GHOAuthPayload.self) else {
```

Comment:
considering the `handle()` method catches this error, i think we're fine with the raw try and no guard